javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -135,8 +135,8 @@ object Long {
   def rotateRight(i: scala.Long, distance: scala.Int): scala.Long =
     (i >>> distance) | (i << -distance)
 
-  def signum(i: scala.Long): scala.Long =
-    if (i < 0L) -1L else if (i == 0L) 0L else 1L
+  def signum(i: scala.Long): Int =
+    if (i < 0L) -1 else if (i == 0L) 0 else 1
```

Comment:
This is a bug :) (a rather serious one, as it will probably lead to type errors at runtime). Could you file this and PR separately. @sjrd I assume we don't have a chance that this makes it into 0.6.2.